lib/components/app/app-frame.js

```diff
@@ -9,7 +9,7 @@ import NotFound from './not-found'
  * content and an optional sub-navigation component can be inserted.
  */
 const AppFrame = ({ children, SubNav }) => (
-  <div className='otp'>
+  <div className='otp' id='otp' role='main'>
```

Comment:
Why does it need an `id`?

Reply:
The rule stated that the aria main element also needs an id to guarantee uniqueness

Reply:
There is a `<main>` element rendered somewhere in the hierarchy already, is that the element that the rule is looking for?

Reply:
I didn't know that. But I think it would have to be set as `<main role='main' id='main'>`. I think the rules are designed to ensure things work on as many screen readers as possible. I guess this is a question for @evansiroky should we move the `role='main'` to the `<main>` element?